introduction.md

```diff
@@ -0,0 +1 @@
+Hey my name is Ryan Haarmann and this is my introduction to the GitHub Mastery Dojo.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.